packages/features/bookings/Booker/components/HavingTroubleFindingTime.tsx

```
@@ -0,0 +1,44 @@
+import { ArrowRight, InfoIcon } from "lucide-react";
+import { useState } from "react";
+
+import { BOOKER_NUMBER_OF_DAYS_TO_LOAD } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+
+type Props = {
+  onButtonClick: () => void;
+  dayCount: number | undefined;
+  isScheduleLoading: boolean;
+};
+export function HavingTroubleFindingTime(props: Props) {
+  const { t } = useLocale();
+  const [internalClick, setInternalClick] = useState(false);
+
+  // Eaiest way to detect if its not enabled
+  if (
+    (process.env.NEXT_PUBLIC_BOOKER_NUMBER_OF_DAYS_TO_LOAD == "0" && BOOKER_NUMBER_OF_DAYS_TO_LOAD == 0) ||
+    !process.env.NEXT_PUBLIC_BOOKER_NUMBER_OF_DAYS_TO_LOAD
+  )
+    return null;
```

If default of "0" is set OR if there is no next public var hide this component